app.js

```diff
@@ -183,12 +188,16 @@ function renderChart() {
   });
 }
 
-function setAllProducts (){
-  let stringify = JSON.stringify(allProducts);
-  localStorage.setItem('All Products', stringify);
+function setItems (){
+  let stringifyVersion = JSON.stringify(allProducts);
+  localStorage.setItem('All Products', stringifyVersion);
+}
 
-  localStorage.getItem('All Products');
-  JSON.parse(stringify);
+function getItems (){
+  let stringifyVersion = localStorage.getItem('All Products');
+  let parsedVersion = JSON.parse(stringifyVersion);
+  
+  return parsedVersion;
```

Comment:
The return is useful if we think we're going to want to use this function multiple times, and assign the return to multiple different values. Otherwise, it's fine to set it to what we want it to be equal to in the function.